swagger.yaml

```diff
@@ -511,6 +531,17 @@ components:
           type: string
           format: date
 
+    Slack-avatar:
+      type: object
+      description: Project data time entries from Forecast API
+      properties:
+        image_original:
+          type: string
+        image_24:
+          type: string
+        image_32:
+          type: string
```

Comment:
Do we need 3 image formats? Could this response contain all of the slack users images and some way of identifying which user an image is associated with.
Not directly related to this but is it worth thinking about storing the images in local storage in the UI. Probably not, but at least it would be nice to only need to make the request once for all users rather than for each user.